base/docs/helpdb/Dates.jl

```diff
@@ -22,11 +22,12 @@ the new `DateTime` are assumed to be zero.
 """
 Dates.DateTime(dt::Date)
 
+
 """
     DateTime(dt::AbstractString, format::AbstractString; locale="english") -> DateTime
 
 Construct a `DateTime` type by parsing the `dt` date string following the pattern given in
```

Comment:
Also, but I found it a bit too much for a small object like a date. Maybe that's just me. ;-)